napari_clusters_plotter/_plotter.py

```diff
@@ -155,36 +170,121 @@ def manual_clustering_method(inside):
                 self.run(
                     features,
                     self.plot_x_axis_name,
                     self.plot_y_axis_name,
                     plot_cluster_name=clustering_ID,
                 )
 
+            except AttributeError:
+                # In this case, replotting is not yet possible
+                pass
+
+        def checkbox_status_changed():
+            replot()
+
+        def plotting_type_changed():
+            replot()
+
+            if self.plotting_type.currentText() == PlottingType.HISTOGRAM_2D.name:
+                self.bin_number_container.setVisible(True)
+                self.log_scale_container.setVisible(True)
+            else:
+                self.bin_number_container.setVisible(False)
+                self.log_scale_container.setVisible(False)
+
+        def bin_number_set():
+            replot()
+
+        def bin_auto():
+            self.bin_number_manual_container.setVisible(not self.bin_auto.isChecked())
+            if self.bin_auto.isChecked():
+                replot()
+
+        # Combobox with plotting types
+        combobox_plotting_container = QWidget()
+        combobox_plotting_container.setLayout(QHBoxLayout())
+        combobox_plotting_container.layout().addWidget(QLabel("Plotting type"))
+        self.plotting_type = QComboBox()
+        self.plotting_type.addItems(
+            [PlottingType.SCATTER.name, PlottingType.HISTOGRAM_2D.name]
+        )
+        self.plotting_type.currentIndexChanged.connect(plotting_type_changed)
+        combobox_plotting_container.layout().addWidget(self.plotting_type)
+
+        self.bin_number_container = QWidget()
+        self.bin_number_container.setLayout(QHBoxLayout())
+        self.bin_number_container.layout().addWidget(QLabel("Number of bins:"))
+
+        self.bin_number_manual_container = QWidget()
+        self.bin_number_manual_container.setLayout(QHBoxLayout())
+        self.bin_number_spinner = QSpinBox()
+        self.bin_number_spinner.setMinimum(1)
+        self.bin_number_spinner.setMaximum(1000)
+        self.bin_number_spinner.setValue(400)
+
+        self.bin_number_manual_container.layout().addWidget(self.bin_number_spinner)
+        self.bin_number_set = QPushButton("Set")
+        self.bin_number_set.clicked.connect(bin_number_set)
+        self.bin_number_manual_container.layout().addWidget(self.bin_number_set)
+
+        self.bin_number_container.layout().addWidget(self.bin_number_manual_container)
+
+        self.bin_auto = QCheckBox("Auto")
+        self.bin_auto.setChecked(True)
+        self.bin_auto.stateChanged.connect(bin_auto)
+        self.bin_number_container.layout().addWidget(self.bin_auto)
+
+        self.bin_number_manual_container.setVisible(False)
+        self.bin_number_container.setVisible(False)
+
+        self.log_scale_container = QWidget()
+        self.log_scale_container.setLayout(QHBoxLayout())
+        self.log_scale_container.layout().addWidget(QLabel("Log scale:"))
```

Comment:
```suggestion
        self.log_scale_container.layout().addWidget(QLabel("Log scale"))
```
Minor change for consistency